operator/Makefile

```diff
@@ -236,20 +251,25 @@ CRDS := centrals.platform.stackrox.io securedclusters.platform.stackrox.io
 validate-crs:
 	./tests/scripts/validate-crs.sh ./tests $(CRDS)
 
+.PHONY: test-e2e
 test-e2e: build install validate-crs kuttl ensure-rox-main-image-exists ## Run e2e tests with local manager.
 	mkdir -p $(PROJECT_DIR)/build/kuttl-test-artifacts
-	KUTTL=$(KUTTL) ENABLE_WEBHOOKS=$(ENABLE_WEBHOOKS) $(KUTTL) test
+	KUTTL=$(KUTTL) REAL_KUBECONFIG=$${KUBECONFIG} ENABLE_WEBHOOKS=$(ENABLE_WEBHOOKS) $(KUTTL) test
+
```

Comment:
```suggestion
```

Reply:
I made it a draft because there are more changes to come, and I was also waiting to see if I can merge this: https://github.com/stackrox/stackrox/pull/2784

The migration to v1.17 include go version 1.17 in go.mod, and I wasn't sure I could do that, due to some downstream problems. But now it's done.